db/migrate/20170816150506_change_division_id_in_loan_question.rb

```diff
@@ -0,0 +1,18 @@
+class ChangeDivisionIdInLoanQuestion < ActiveRecord::Migration
+  def up
+    # Change column from string to integer
+    LoanQuestion.all.each { |q| q.update_attribute(:division_id, nil) }
+    change_column :loan_questions, :division_id, :integer, using: 'division_id::integer'
+    LoanQuestion.all.each { |q| q.update_attribute(:division_id, 99) }
+
+    change_column_null :loan_questions, :division_id, false
+  end
+
+  def down
+    # Change column from integer to string
+    change_column :loan_questions, :division_id, :string
+    LoanQuestion.all.each { |q| q.update_attribute(:division_id, 99) }
+
+    change_column_null :loan_questions, :division_id, true
+  end
```

Comment:
Yes, it was a mistake in the first migration. Is it best to just adjust all these migrations so that the string to integer conversion does not ever happen? Thanks.